src/library_sdl.js

```diff
@@ -2209,7 +2209,7 @@ var LibrarySDL = {
 
       if (!raw) {
         filename = PATH_FS.resolve(filename);
-        raw = preloadedImages[filename];
+        raw = Browser.preloadedImages[filename];
```

Comment:
Missing dependency on $Browser? (and the other spots in this file)

Reply:
This is also handled transitively I think sine we have `autoAddDeps(LibrarySDL, '$SDL');` below and SDL depends on `Browser.`   Maybe I should make this one explicit though.